rclcpp/include/rclcpp/strategies/allocator_memory_strategy.hpp

```diff
@@ -392,16 +389,18 @@ class AllocatorMemoryStrategy : public memory_strategy::MemoryStrategy
     rclcpp::AnyExecutable & any_exec,
     const WeakCallbackGroupsToNodesMap & weak_groups_to_nodes) override
   {
-    auto it = waitable_handles_.begin();
-    while (it != waitable_handles_.end()) {
+    auto & waitable_handles =
```

Comment:
This seems to be a trick to make the `TEST_F(TestAllocatorMemoryStrategy, get_next_waitable)` passed.

Reply:
Yes. Maybe need to re-write get_next_waitable.